python/test/unit/runtime/test_bindings.py

```diff
@@ -79,3 +80,26 @@ def walk_fn(op):
 
     ttir_module = src.make_ir(options, codegen_fns, context)
     ttir_module.walk(walk_fn)
+
+
+def test_python_func_in_visit_call(device):
+
+    @triton.jit
+    def test_py_call_const_kernel(
+        in_ptr0,
+        out_ptr,
+        n_elements,
+        BLOCK_SIZE: "tl.constexpr",
+    ):
+        log2e: tl.constexpr = math.log2(math.e)
```

Comment:
I don't think we can support that as it would break caching. Our cachine mechanism would not check whether the callee has changed and therefore would not be able to invalidate the cache if it has.

Reply:
I think when @peterbell10 initialized the check, he meant to check `func` but not `node.func`? So built-in functions are just not considered

Reply:
Agree with Jokeren's opinion, this code seems like some typo error?

Reply:
Sounds good, I'll Keren do the review then